src/main/java/racinggame/view/ResultView.java

```diff
@@ -1,16 +1,58 @@
 package racinggame.view;
 
+import racinggame.domain.Car;
+import racinggame.domain.Position;
 import racinggame.domain.RaceRecord;
 
+import java.util.ArrayList;
 import java.util.List;
 
 public class ResultView {
 
-    private ResultView() {}
+    private static final String WINNER_SEPARATOR = ", ";
+    private ResultView() {
```

Comment:
상수와 생성자의 구분을 위해 공백 라인을 추가하면 좋을 것 같습니다.
```suggestion
    private static final String WINNER_SEPARATOR = ", ";
    
    private ResultView() {
```